notebooks/concepts/aida.ipynb

```diff
@@ -0,0 +1,145 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5bd4b7fa-d7ed-48e8-92bc-442ed336ec23",
+   "metadata": {},
+   "source": [
+    "# Shared API for viewer orientation\n",
+    "\n",
+    "#### Load observations of the Cartwheel Galaxy into `imviz`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ad82c2ef-be3a-4316-a82d-c11ede32064d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from jdaviz import Imviz\n",
+    "\n",
+    "viz = Imviz()\n",
+    "viz.load_data('../../jdaviz/notebooks/jw02727-o002_t062_nircam_clear-f277w_i2d.fits')\n",
+    "viz.show(height=400)\n",
+    "\n",
+    "imviz_viewer = viz.app.get_viewer(\"imviz-0\")\n",
```

Comment:
are we making this public yet?  If so, it would be nice to expose to `viz.viewers['imviz-0']` and use that in any notebooks/tests.

Reply:
I put this in concepts because I wasn't making it public yet. If you're ok with it, this is part of the follow-up [ticket](https://jira.stsci.edu/browse/CST-315) to make this all public and deprecate the old astrowidgets api.

Reply:
ok, then maybe we should hold off on the changelog entry and can add the changelog as part of that.  There may very well be a 4.3 release between this getting merged and it becoming public API.